assets/sass/v2/_identify.scss

```diff
@@ -23,10 +23,17 @@
       margin-bottom: 0.83em;
     }
     .button {
-      display: block;
+      display: flex;
       position: relative;
-      height: 50px;
-      line-height: 50px;
+      min-height: 60px;
```

Comment:
With this change, in the device with bigger screen that only one line is displayed , the button's height is 60px instead of 50px. Is this expected, @stevennguyen-okta ?

Reply:
That is not expected, preferably the one-line button would match its existing height and only change to accommodate the button wrap.

Reply:
made changes.
one line button is 50px now